opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -388,19 +389,111 @@ namespace Opm {
         }
     }
 
+    Opm::Value<int> Schedule::getValueItem(DeckItemPtr item){
```

Comment:
This *highly* specialized function should not be part of `Schedule` class, but rather be a free function with static linkage. The input argument can be of type `DeckIntItemPtr`.